pkg/agent/secondarynetwork/init_windows.go

```diff
@@ -25,22 +25,30 @@ import (
 	"k8s.io/client-go/tools/cache"
 	componentbaseconfig "k8s.io/component-base/config"
 
+	"antrea.io/antrea/pkg/agent/secondarynetwork/podwatch"
 	agentconfig "antrea.io/antrea/pkg/config/agent"
 	"antrea.io/antrea/pkg/util/channel"
 )
 
-func Initialize(
+type Controller struct {
+	PodController *podwatch.PodController
```

Comment:
why adding this?

Reply:
We need to call Controller.PodController.Run() in `cmd/antrea-agent/agent.go` after exposing `PodController` as public. If  I don't add this property in init_windows.go, it returns compile error on Windows.

Reply:
It's a bit strange to call a struct's member's method, I think you should keep the `Controller`'s Run method.

When I suggested to make `podController` public, I mean making it public so `pkg/agent/secondary` can use call the struct's method directly, instead of having to relying on another interface. I don't mean to calling podController's method directly in pkg/cmd directly.

Reply:
I see, I would update.